src/gt4py/ir/nodes.py

```diff
@@ -278,9 +278,11 @@ def __str__(self):
 
 
 @enum.unique
-class AccessIntent(enum.Enum):
-    READ_ONLY = 0
-    READ_WRITE = 1
+class AccessIntent(enum.IntFlag):
```

Comment:
Is there any problem or incompatibility to import and use the `AccessKind` enum defined in `definitions.py` instead of defining a equivalent one here?